src/DynamoCoreWpf/Services/UsageReportingManager.cs

```diff
@@ -46,13 +46,11 @@ public static void DestroyInstance()
         /// </summary>
         public bool IsUsageReportingApproved
```

Comment:
what is UsageReporting again? Is this ADP? GA? Something else?

Reply:
Will check again.